examples/01_plotting/plot_demo_plotting.py

```diff
@@ -1,19 +1,22 @@
 """
-Plotting in nilearn
+Plotting tools in nilearn
 ==========================
 
-Nilearn comes with a set of plotting function for Nifti-like images,
-see :ref:`plotting` for more details.
-"""
+Nilearn comes with a set of plotting functions for easy visualization of
+Nifti-like images such as statistical maps overlayed on anatomical
+images or overlayed on glass brain image, anatomical images, functional/EPI
+images, region specific mask images.
 
-# Import plotting and image processing tools
-from nilearn import plotting, image
+See :ref:`plotting` for more details.
+"""
 
 ###############################################################################
-# Retrieve the data: haxby dataset to have EPI images and masks, and
-# localizer dataset to have contrast maps
+# First, we retrieve the data from nilearn: haxby dataset to have EPI images
```

Comment:
the data does not exactly belong to nilearn. perhaps "retrieve from nilearn-provided (general-purpose) datasets"